### ISSUE TITLE
ImageMagick@7.1.0-37 is broken (404)

### ISSUE THREAD

**Issue by ogerardin** [ScoopInstaller/Main#3699]
## Bug Report

ImageMagick

### Current Behaviour
`scoop install imagemagick` fails with 404 error:
```
Installing 'imagemagick' (7.1.0-37) [64bit]
The remote server returned an error: (404) Not Found.
URL https://download.imagemagick.org/ImageMagick/download/binaries/ImageMagick-7.1.0-37-Q16-HDRI-x64-dll.exe is not valid

```

### Expected Behaviour

Install succeeds

### Possible Solution
URL pattern seems to have changed, latest download link from https://imagemagick.org/script/download.php is: https://imagemagick.org/archive/binaries/ImageMagick-7.1.0-39-Q16-HDRI-x64-dll.exe

### System details

n/a

#### Scoop Configuration

n/a

**Comment by rashil2000** [comment]
Hi! Can you please make a PR?

**Comment by krystofkrticka** [comment]
It looks like something broke in autoupdate script because when I looked into json of imagemagick it should autoupdate.

**Comment by krystofkrticka** [comment]
Ok it looks like I found the issue they changed domain name for download archive.

**Comment by krystofkrticka** [comment]
@rashil2000 OK, PR is done hopefully it's correct.

**Comment by rashil2000** [comment]
Thanks for the PR! Could you also do this for https://github.com/ScoopInstaller/Main/blob/master/bucket/imagemagick-lean.json?

**Comment by krystofkrticka** [comment]
OK, I'll give it a look.

**Comment by krystofkrticka** [comment]
> Thanks for the PR! Could you also do this for https://github.com/ScoopInstaller/Main/blob/master/bucket/imagemagick-lean.json?

Fixed in PR: #3704
